torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -190,11 +190,15 @@ std::string valueName(Value * n) {
   return "n" + std::to_string(n->unique());
 }
 
- std::string scalarValue(const at::Tensor & t) {
+std::string scalarValue(const at::Tensor & t) {
   auto s =  at::Scalar(t);
-  return (s.isIntegral()) ?
-    std::to_string(s.toLong()) :
-    (std::to_string(s.toDouble()) + "f");
+  if (s.isIntegral()) 
```

Comment:
nit: if you're using braces for one of the branches, can you please add it to the other branch as well?